doc/source/whatsnew/v0.19.0.txt

```diff
@@ -437,6 +437,7 @@ API changes
 - ``__setitem__`` will no longer apply a callable rhs as a function instead of storing it. Call ``where`` directly to get the previous behavior. (:issue:`13299`)
 - Passing ``Period`` with multiple frequencies to normal ``Index`` now returns ``Index`` with ``object`` dtype (:issue:`13664`)
 - ``PeriodIndex.fillna`` with ``Period`` has different freq now coerces to ``object`` dtype (:issue:`13664`)
+- Faceted boxplots (from ``DataFrame.groupby(col).boxplot`` or ``DataFrame.boxplot(by=col))`` now return a ``Series``. Previously the groupby version returned an ``OrderedDict``, while the by version returned a NumPy Array (:issue:`12216`, :issue:`7096`)
```

Comment:
... except for ``DataFrame.boxplot(by=col, return_type='axes')`` which still returns an array